underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java

```diff
@@ -79,17 +78,13 @@
    *
    * @param bucketName the name of the bucket
    * @param conf the configuration for Alluxio
+   * @param awsCredentials AWS Credentials configuration for S3 Access
    * @throws ServiceException when a connection to S3 could not be created
    */
-  public S3UnderFileSystem(String bucketName, Configuration conf) throws ServiceException {
+  public S3UnderFileSystem(String bucketName, Configuration conf,
+                           AWSCredentials awsCredentials) throws ServiceException {
```

Comment:
Could you make the formatting consistent with the codebase here? (args one line, new line for throws)